### PR TITLE
Merge 'Development' into 'main'

### PR DESCRIPTION
**Pull request by GhausAliKhan**
**In this branch we have:**

- Incorporated extensive styling following design guidelines, including color schemes, typography, and layout.
- Modularized styles by creating separate CSS files for each component.
- Implemented features like a price graph that shows crypto prices since last year and 'Back' button for better navigation, which clears selected cryptocurrency details and resets active search filters.
- Developed unit and integration tests to ensure the application's robustness and reliability.
- Improved interaction on both Home and Details pages, fully aligning with project requirements.

**Conclusion**
This pull request seeks to merge the fully developed and tested 'develop' branch into the 'main' branch. It represents the completion of the "My Crypto Metrics" capstone project. Future work may involve minor enhancements or bug fixes, but the core functionalities are stable and complete.

### CODE REVIEW

File: src/styles/cryptolist.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.coinCard {
+  width: 50%;
+  background-color: #b24364;
+  border: 1px solid #fff;
+  padding: 60px 60px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (Strangeal) [inline src/styles/cryptolist.css]: - [ ] Great job on styling your project so beautifully! 👏 However, please note that the project requirements specifically state that you should personalize the project design while maintaining its `color`, `layout`, and `Typography`. 

Screenshot

   - I can see that you've done a fantastic job maintaining the design `layout` and `Typography`, but it looks like the `color` needs some alignment with the project's requirements. To fully adhere to the project's design specifications, please make sure to select one of the colors provided in the design and use it consistently in your project styles. Here's a reference to the available colors you can choose from:

Color Palette

Keep up the excellent work! 🎨✨

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @GhausAliKhan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Testing ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `vercel` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Well structured project ✅ 

## Required Changes ♻️
 - [ ] Great job on your project presentation! 🚀 Your presentation is clear and professional, which is a great start. However, I noticed one key element that could enhance your presentation further: **showcasing a piece of code that you're particularly proud of**. This element is crucial as it allows you to highlight the part of the project that you're most passionate about and provides your audience with a deeper understanding of your work.

    - To make your presentation even more engaging, consider re-recording the video to include this important key point. Sharing a piece of code that you're proud of not only demonstrates your technical skills but also adds a personal touch to your presentation, making it more relatable to your audience.

Keep up the excellent work! 😊
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/cryptolist.css (hunk 1 above)
Comment (GhausAliKhan) [reply src/styles/cryptolist.css]: Thankyou @Strangeal  for your clear instructions. I've added the changes 💯

**Review by frank1738 — APPROVED** [review]
Hi @GhausAliKhan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Application meets the design guidelines
✅ Application is well-tested



## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**